schema/spacewalk/common/tables/rhnPackage.sql

```diff
@@ -56,6 +56,7 @@ CREATE TABLE rhnPackage
                          CONSTRAINT rhn_package_compat_check
                              CHECK (compat in ( 1 , 0 )),
     path             VARCHAR(1000),
+    remote_path	     VARCHAR(1000),
```

Comment:
Is there a reason, except 'we did it like that before', to limit path to 1000 characters?
In postgresql varchar is implemented using TEXT datatype with string length check on top of it. So using limited VARCHAR is actually a tiny little slower then just using TEXT and has no space or indexing benefit.

So if there is no reason to limit this to 1000 characters, please use TEXT as a datatype here.

Reply:
Absolutely..thanks for the review,
I did not have any specific reason for using `VARCHAR(1000)`...I'll change it to `TEXT` instead.